.github/workflows/maven.yml

```diff
@@ -38,7 +38,24 @@ jobs:
         key: ${{ runner.os }}-sonar
         restore-keys: ${{ runner.os }}-sonar
     - name: Build
-      run: ./mvnw --no-transfer-progress -V -B -ff -s .github/settings.xml install
+      run: ./mvnw --no-transfer-progress -V -B -ff -s .github/settings.xml -e -DtrimStackTrace=false -Dmaven.test.failure.ignore=true -Dsurefire.rerunFailingTestsCount=1 verify
+    - name: Publish Test Results
+      # If the CI run is not initiated from the primary repository, it is highly likely that this is a PR from a user who doesn't have commit rights.
+      # Hence, skip this step to avoid permission failures.
+      if: github.repository == 'dropwizard/dropwizard'
+      uses: scacap/action-surefire-report@v1
+      with:
+        github_token: ${{ secrets.GITHUB_TOKEN }}
+        check_name: Test Report (${{ matrix.os }} - Java ${{ matrix.java_version }})
+        report_paths: '**/*-reports/TEST-*.xml'
+    - name: Upload Test Reports
+      # If the CI run is not initiated from the primary repository, it is highly likely that this is a PR from a user who doesn't have commit rights.
+      # Hence, skip this step to avoid permission failures.
+      if: github.repository == 'dropwizard/dropwizard'
```

Comment:
Looks like this isn't doing what you expected.

Google led me to this as a way of detecting PRs from forks - what do you think?

https://github.community/t/how-to-detect-a-pull-request-from-a-fork/18363/3

Reply:
@rhowe Seems to work that way. Thanks a lot! 😄